pkg/auth/jwks.go

```diff
@@ -0,0 +1,49 @@
+package auth
```

Comment:
Is auth the right package for this? Looks like it could be used for other purposes than authentication.

Maybe just `pkg/jwks/jwks.go` or `pkg/utils/jwks.go`